DFC.FindACourseClient/Models/APIRequests/CourseSearchRequest.cs

```diff
@@ -37,5 +37,70 @@ public class CourseSearchRequest
         public double? Longitude { get; set; }
 
         public string CampaignCode { get; set; }
+
+        public bool ShouldSerializeDistance()
+        {
+            return (Longitude != null && Latitude != null) || !string.IsNullOrEmpty(Postcode);
+        }
+
+        public bool ShouldSerializeProviderName()
```

Comment:
Where do all these end up getting used? Just checking as no new views in here or anything

Reply:
Ah sorry just saw the comment - its a library via nuget package so makes sense now